builder/dockerfile/internals.go

```diff
@@ -188,8 +190,6 @@ func (b *Builder) runContextCommand(args []string, allowRemote bool, allowLocalD
 	}
 	b.tmpContainers[container.ID] = struct{}{}
 
-	comment := fmt.Sprintf("%s %s in %s", cmdName, origPaths, dest)
```

Comment:
This comment was not actually being used. The branch that required comment was only hit if `id == ""`, but we call commit with an ID here.

Removing this meant that `origPaths` was also no longer used.  But it also revealed that the two comments were using different values. One ignored comment string used `origPaths` , the actual comment string uses `srcHash`.

I'm not sure which one is correct. I've left the old behaviour for now, with a TODO about using `origPaths`. If anyone know which is actually correct I can remove the TODO.